admin/app/templates/authenticated/users/get/certification-center-memberships.hbs

```diff
@@ -1,3 +1,6 @@
 <section class="page-section">
-  <Users::CertificationCenterMemberships @certificationCenterMemberships={{@model.certificationCenterMemberships}} />
+  <Users::CertificationCenterMemberships
+    @certificationCenterMemberships={{@model.certificationCenterMemberships}}
+    @disableCertificationCenterMembership={{this.disableCertificationCenterMembership}}
```

Comment:
**remarque**
cette ligne ne sert plus `@disableCertificationCenterMembership={{this.disableCertificationCenterMembership}}` tu peux l'enlever :)

Reply:
Done !